### ISSUE TITLE
feature request: allow puppeteer options to be customized.

### ISSUE THREAD

**Issue by jariz** [alvarlagerlof/next-banner#84]
Currently, it the CLI crashes because [puppeteer crashes with the default settings on github actions](https://stackoverflow.com/questions/62228154/puppeteer-fails-to-initiate-in-github-actions). 
Stackoverflow tells me that `headless` should be `true`, but I have no way to do this besides modifying the CLI itself.
So, it would be nice to be able to pass additional puppeteer configuration.
I will likely open a PR for this.

**Comment by alvarlagerlof** [comment]
Good idea. I changed the default to `headless: true` in https://github.com/alvarlagerlof/next-banner/releases/tag/v1.1.7. Seems reasonable. 

About the other puppeteer options, they are currently passed through json once. Would what be a problem for the options you want to use or not? I could include an object that gets merged with the current options in `withNextBanner` in `next.config.js`
